client/src/app.jsx

```diff
@@ -86,7 +86,7 @@ class App extends React.Component {
     //get the propertyID from this url
     // const propID = url[]
     //send GET to /photos/${propID}
-    axios.get(`/photos/1`)
+    axios.get(`/photos/10`)
```

Comment:
looks like you still have this hardcoded. will need to request based on id in the url

Reply:
Ok, got it. Thanks for the callout, I'll make sure to go change that once I have it running on my proxy. I'll make a ticket for it